src/style_guide.css

```diff
@@ -0,0 +1,232 @@
+html * {
```

Comment:
I think the same thing could be accomplished using body { color: #5B34B6 }

html * is a rather expensive selector because the browser would evaluate every element (evaluation moves right to left), then retain the elements that are a descendant of html

Give this a glance https://csswizardry.com/2011/09/writing-efficient-css-selectors/